web/src/pages/subscription-plans/components/current-subscription/index.jsx

```diff
@@ -27,11 +26,11 @@ const CurrentSubscriptionBlock = ({ onCancelSubscription, currentSubscription })
         <Text size="sm">
           Your subscription started on
           {' '}
-          {dayjs(currentSubscription.startDate).format('MMM DD, YYYY')}
+          {dayjs(new Date(currentSubscription.startDate * 1000)).format('MMM DD, YYYY')}
           {' '}
-          and expires
+          and renews
           {' '}
-          {dayjs(currentSubscription.endDate).format('MMM DD, YYYY')}
+          {dayjs(new Date(currentSubscription.endDate * 1000)).format('MMM DD, YYYY')}
```

Comment:
Please create variables for start/end subscription dates and use string interpolation here